--- hfst-ospell-0.5.3/hfst-ol.h.orig	2022-03-13 11:48:36.000000000 +0100
+++ hfst-ospell-0.5.3/hfst-ol.h	2023-06-29 19:15:49.581959666 +0200
@@ -26,6 +26,7 @@
 #include <vector>
 #include <map>
 #include <climits>
+#include <cstdint>
 #include <cstdio>
 #include <cstdlib>
 #include <iostream>
--- hfst-ospell-0.5.3/office.cc.orig	2022-03-13 11:48:36.000000000 +0100
+++ hfst-ospell-0.5.3/office.cc	2023-06-29 19:16:08.511397682 +0200
@@ -42,6 +42,7 @@
 #include <cmath>
 #include <cerrno>
 #include <cctype>
+#include <cstdint>
 #include <getopt.h>
 
 #define U_CHARSET_IS_UTF8 1
