--- hfstospell-0.4.0/main-fsmnlp-2012.cc.orig	2012-09-04 14:43:05.000000000 +0200
+++ hfstospell-0.4.0/main-fsmnlp-2012.cc	2015-12-28 19:01:30.869303121 +0100
@@ -255,15 +255,7 @@ zhfst_spell(char* zhfst_filename)
     {
       std::cerr << "cannot read zhfst archive " << zhfst_filename << ":" 
           << std::endl
-          << zhzre.what() << "." << std::endl
-          << "trying to read as legacy automata directory" << std::endl;
-      speller.read_legacy(zhfst_filename);
-    }
-  catch (hfst_ol::ZHfstLegacyReadingError zhlre)
-    {
-      std::cerr << "cannot read legacy hfst speller dir " << zhfst_filename 
-          << ":" << std::endl
-          << zhlre.what() << "." << std::endl;
+          << zhzre.what() << "." << std::endl;
       return EXIT_FAILURE;
     }
 
--- hfstospell-0.4.0/main-ispell.cc.orig	2014-04-28 11:05:20.000000000 +0200
+++ hfstospell-0.4.0/main-ispell.cc	2015-12-28 18:51:52.375994063 +0100
@@ -279,20 +279,8 @@ zhfst_spell(char* zhfst_filename, FILE*
     catch (hfst_ol::ZHfstZipReadingError zhzre)
       {
         std::cerr << "cannot read zhfst archive " << zhfst_filename << ":" 
-          << zhzre.what() << "." << std::endl
-          << "trying to read as legacy automata directory" << std::endl;
-        try 
-          {
-            speller.read_legacy(zhfst_filename);
-          }
-        catch (hfst_ol::ZHfstLegacyReadingError zhlre)
-          {
-            std::cerr << "cannot fallback to read legacy hfst speller dir " 
-              << zhfst_filename 
-              << ":" << std::endl
-              << zhlre.what() << "." << std::endl;
+          << zhzre.what() << "." << std::endl;
             return EXIT_FAILURE;
-          }
       }
     if (verbose)
       {
